src/mcmcchains-connect.jl

```diff
@@ -0,0 +1,32 @@
+import .MCMCChains: Chains
+
+# A basic chains constructor that works with the Transition struct we defined.
+function AbstractMCMC.bundle_samples(
+    ts::Vector{<:Transition},
+    model::AbstractMCMC.AbstractModel,
+    sampler::AbstractMCMC.AbstractSampler,
+    state,
+    chain_type::Type{Chains};
+    discard_initial=0,
+    thinning=1,
+    param_names=missing,
+    bijector=identity,
+    kwargs...
+)
+    # Turn all the transitions into a vector-of-vectors.
+    vals = [vcat(bijector(t.z.θ), t.z.ℓπ.value) for t in ts]
```

Comment:
It's important to return the other statistics as well:
```suggestion
    tstat = merge((; lp = t.z.ℓπ.value), stat(t))
    tstat_names = collect(keys(tstat))
    vals = [vcat(bijector(t.z.θ), collect(tstat)) for t in ts]
```